apps/hash-frontend/src/pages/[shortname]/types/entity-type/[...slug-maybe-version].page/entity-type-description.tsx

```diff
@@ -0,0 +1,34 @@
+import { EditableField } from "@hashintel/block-design-system";
+import { EntityTypeEditorFormData } from "@hashintel/type-editor";
+import { Typography } from "@mui/material";
+import { useController, useFormContext } from "react-hook-form";
+
+interface EntityTypeDescriptionProps {
+  readonly?: boolean;
+}
+
+export const EntityTypeDescription = ({
+  readonly,
+}: EntityTypeDescriptionProps) => {
+  const { control } = useFormContext<EntityTypeEditorFormData>();
+
+  const descriptionController = useController({
+    control,
+    name: "description",
+  });
+
+  return (
+    <>
+      <Typography variant="h5" sx={{ mb: 1, fontWeight: 500 }}>
+        Description
+      </Typography>
+
```

Comment:
Let's remove this as David suggests

Reply:
Done in b6558f827674f7447c5d807aed6fa04d7afce5f2